.annotations_features

```diff
@@ -0,0 +1,19 @@
+source_path: ../
+report_path: reports
+safelist_path: .annotation_safe_list.yml
+coverage_target: 50.0
+annotations:
+    ".. feature_toggle_name:":
+    ".. feature_toggle_type:":
+    ".. feature_toggle_default:":
+    ".. feature_toggle_type:":
+    ".. feature_toggle_description:":
+    ".. feature_toggle_category:":
+    ".. feature_toggle_use_cases:":
```

Comment:
I'm not sure if we want to specify choices here. [OEP17](https://open-edx-proposals.readthedocs.io/en/latest/oep-0017-bp-feature-toggles.html#use-cases) enumerates the following:

- Use Case 1: Incremental Release
- Use Case 2: Launch Date
- Use Case 3: Ops - Monitored Rollout
- Use Case 4: Ops - Graceful Degradation
- Use Case 5: Beta Testing
- Use Case 6: VIP / White Label
- Use Case 7: Opt-out
- Use Case 8: Open edX option  

If these are useful / not too confusing, we could go with: 

```suggestion
    ".. feature_toggle_use_cases:":
        choices: [incremental_release, launch_date, monitored_rollout, graceful_degradation, beta_testing, vip, opt_out, open_edx]
```